fakebook/src/environments/environment.ts

```diff
@@ -4,8 +4,10 @@
 
 export const environment = {
   production: false,
-  baseUrl: 'someUrl' // replace with backend development url
-
+  baseUrl: 'http://localhost:4200',
+  profile: {
+    url: 'http://localhost:4200'
+  }
```

Comment:
this probably isn't useful since the local angular app is probably running at 4200, probably not any backends.

one of the services is using the `baseUrl`, the other is using `profile.url`. this isn't very maintainable.

either way ensure the production environment file is consistent, that's crucial.